packages/data-stores/src/wpcom-plans-ui/actions.ts

```diff
@@ -24,6 +24,15 @@ export const setSelectedStorageOptionForPlan = ( {
 		planSlug,
 	} ) as const;
 
+export const setSelectDropdownSelectorPlanOrder = ( { planOrder }: { planOrder: PlanSlug[] } ) =>
```

Comment:
Maybe we can be a little more specific about this. `setSelectDropdownSelectorPlanOrder` comes across as a little vague and unclear to me. I can't follow what it's referring to - just spelling it out: _"set the select dropdown selector plan order"_? 🤔 

Maybe I'm missing something. This is just the comparison-grid's plans order though, right? Is there anything else to it? If not, then maybe `WPCOM_PLANS_UI_SET_COMPARISON_GRID_PLANS_ORDER` might be simple/accurate enough

Reply:
sorry for early comment though if this is still in an early/experimental state :-)

Reply:
Cool. I follow the general idea. I'll take a closer look a little later and give you my thoughts :-)